### DISCUSSION TITLE
vps部署的alist是否可以通过局域网alist套娃vps的alist实现一键本地代理？

### DISCUSSION THREAD

**Discussion by CODPHISH** [AlistGo/alist#7537] [Ideas]
比如vps上有的驱动是302，有的驱动是本地，但局域网的alist套娃vps上的后，所有的驱动都由局域网alist接管，并且改为本地代理，这样播放就不会消耗vps的流量，同时还能远程vps的alist管理文件

**Comment by yinaoxiong** [comment]
感觉现在这种套娃如果要是必须要是本地代理的估计还是走vps的流量。（没看源码不知道我理解的对不

感觉这种需求可以通过类似存储配置同步来实现，vps的是中心服务器， 局域网可以从上游同步存储配置。
